### ISSUE TITLE
Make neural network interface both for theano and tensorflow

### ISSUE THREAD

**Issue by thechaos16** [thechaos16/Machine_Learning#15]
1. Standard neural network
- [ x ] keras
- [ ] tensorflow (WIP)

2. Convolutional neural network
- [ ] keras
- [ ] tensorflow

3. LSTM / recurrent neural network
- [ ] keras
- [ ] tensorflow

**Comment by thechaos16** [comment]
For Batch normalization in keras, parameter ```activation``` for Dense should be ```linear```, and add BatchNormalization afterwards, and add other activation like PReLU at the end
